### PR TITLE
Database

### PR DESCRIPTION
**Pull request by kko138**

### CODE REVIEW

File: src/Database.java [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
 public class Database {
-	private static final String userName = "user";
-	private static final String passWord = "pw";
+	
+	public static Connection connection;
+	
+	public static ResultSet queryHandler(Map<String, String> map) throws SQLException {
+		ResultSet returnMe = null;
+		try {
+		Class.forName("com.mysql.jdbc.Driver");
```
Comment (john-ko) [inline src/Database.java:11]: tabs pls

File: src/Database.java [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+		connection = DriverManager.getConnection("jdbc:mysql://" + Config.address + "/" + Config.dbname, Config.userName, Config.passWord);
+		
+		Statement statement = null;
+		String sql = "SELECT * FROM products WHERE ";
```
Comment (john-ko) [inline src/Database.java]: update sql

File: src/Database.java [hunk 3]
```diff
@@ -1,5 +1,35 @@
+import java.sql.*;
+import java.util.*;
 
 public class Database {
-	private static final String userName = "user";
-	private static final String passWord = "pw";
```
Comment (sophiech) [inline src/Database.java:4]: hi

File: src/Database.java (hunk 1 above)
Comment (sophiech) [inline src/Database.java:11]: yes tab pls

File: src/Database.java [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+	
+	public static Connection connection;
+	
+	public static ResultSet queryHandler(Map<String, String> map) throws SQLException {
+		ResultSet returnMe = null;	
+		try {
+		Class.forName("com.mysql.jdbc.Driver");
```
Comment (john-ko) [inline src/Database.java:11]: pls fix tabs

**Pull request merged**